cmd/integration_test/build/buildspecs/test-eks-a-cli.yml

```diff
@@ -74,6 +74,11 @@ env:
     T_REGISTRY_MIRROR_USERNAME: "harbor-registry-data:username"
     T_REGISTRY_MIRROR_PASSWORD: "harbor-registry-data:password"
     T_REGISTRY_MIRROR_CA_CERT: "harbor-registry-data:caCert"
+    T_REGISTRY_MIRROR_ENDPOINT_TINKERBELL: "harbor-registry-tinkerbell-data:endpoint"
```

Comment:
No need to create a new secret in secrets manager. Just put these new keys in the tinkerbell_ci secret